Source/WebInspectorUI/UserInterface/Views/RecordingContentView.css

```diff
@@ -49,7 +49,7 @@
 
 .content-view.recording > header > .slider-container > .slider-value {
     font-family: Menlo, monospace;
-    font-family: 11px;
+    font-size: 11px;
```

Comment:
actually i think we'd be better off just removing this entirely since that'd more closely preserve existing behavior